doc/build-freebsd.md

```diff
@@ -98,8 +97,8 @@ pkg install python3
 ### 1. Configuration
 
 There are many ways to configure Bitcoin Core, here are a few common examples:
-##### Wallet (BDB + SQlite) Support, No GUI:
-This explicitly enables legacy wallet support and disables the GUI. If `sqlite3` is installed, then descriptor wallet support will be built.
+##### Wallet & Legacy Wallet Support, No GUI:
+This explicitly enables legacy wallet support and disables the GUI. If `sqlite3` is installed, then wallet support will also be built.
```

Comment:
I am going a little back and forth on the terms `legacy wallet` and `wallet`. It indicates that there are two different kind of wallets in the context, but doesn't explain what's the difference. So may be instead we could use `legacy wallet` and `descriptor wallet` in the doc everywhere? In that way I feel it would be more clear to new readers. Because it is not a well known fact outside of the dev community that a sqlite based default wallet is descriptor based and its not same as regular BDB wallets.